examples/acm-terraform-blog-part3/README.md

```diff
@@ -0,0 +1,88 @@
+# Enable ACM features with Terraform - Part 3
+
+This is Part3 of the tutorial to accompany a short series of  blog articles explaining how to enable [Anthos Config Management (ACM)](https://cloud.google.com/anthos/config-management) with Terraform.
+
+In the [first part](../acm-terraform-blog-part1), we explained how to use Teraform to create a cluster and manage its config from git via [Config Sync](https://cloud.google.com/anthos-config-management/docs/config-sync-overview).
+
+In the [second part](../acm-terraform-blog-part2) we added guard rails for the cluster configuration via [Policy Controller](https://cloud.google.com/anthos-config-management/docs/concepts/policy-controller).
+
+In this article we'll demonstrate how, using Config Connector, you can provision your GCP cloud resources following the same GitOps K8s native model.
+
+## Provision GCP resources
+
+1. Set the variable for the project from [part2](../acm-terraform-blog-part2). We will re-use that project but create a new cluster since we cleaned up at the end of the first section. If you are working in a different project, enable required GCP APIs, as described in [part1/README.md](../part1/README.md).
+
+    ```bash
+    PROJECT_ID = [PROJECT_ID]
+    ```
+1. Note that [wordpress-bundle.yaml](./config-root/wordpress-bundle) was updated to use GCP MySQL database. Also we added [configconnector.yaml](./config-root/configconnector.yaml) to initialize the instance of Config Connector add-on on the cluster.
+
+1. Use [kpt](kpt.dev) to customize the `config-root` directory that will be configured as the source of the objects installed on the cluster.
+
+    ```bash
+    kpt fn eval --image gcr.io/kpt-fn/apply-setters:v0.1 ./config-root -- projectId=$PROJECT_ID
+    ```
+1. Submit the updated configuration into your branch.
+1. Ensure that `sync_repo` and `sync_brach` variables are updated in [variables.tf](./terraform/variables.tf)
```

Comment:
Typically, variable customization should be done in a `terraform.tfvars` file instead of modifying the actual variable declarations.

Reply:
Yes, I understand you want users to change the value. That's the whole point of variables—they should edit it in a tfvars file, not just edit the variable default.

Reply:
My logic was that once the repo is cloned the original values for these specific variables are irrelevant, so there is no value add for customers to carry them over. Although we can say that pointing to this repo informs them of the origin, if we have this pattern with other samples, than I should use tvars.

Reply:
Right, if we ask users to edit a variable it should be done via `terraform.tfvars` (you can provide them a sample to edit from).

Reply:
Changd to use tvars file for all 3 parts.